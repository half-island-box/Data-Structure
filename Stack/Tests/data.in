input data for testing.
(1) 1+2+3+4#
(2) 88-1*5#
(3) 1024/4*8#
(4) 1024/(4*8)#
(5) (20+2)*(6/2)#
(6) 2*(6+2*(3+6*(6+6)))#
(7) (((6+6)*6+3)*2+6)*2#
(8) 8/(9-9)#
