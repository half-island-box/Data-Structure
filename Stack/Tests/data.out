output data for testing
(1) 10
(2) 83
(3) 2048
(4) 32
(5) 66
(6) 312
(7) 312
(8) 0 can't be a denominator.
